src/commands/autocomplete/path.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+
+import AutocompleteBase from '.'
+import path from 'path'
+
+export default class AutocompletePath extends AutocompleteBase {
+  static topic = 'autocomplete'
+  static command = 'commandspath'
+  static description = 'path to autocomplete commands'
+  static hidden = true
+
+  async run () {
```

Comment:
@RasPhilCo you do not have to do this, but you could define `run` in AutocompleteBase which calls another method, maybe `_run` which you define in the subclasses so that they all have the same behavior in them without explicitly calling `errorIfWindows` everywhere